### PR TITLE
clean: Run ruff check --fix on existing code and fix some lines

### PR DESCRIPTION
**Pull request by frikilax**
Code was cleaned using "safe" ruff linting validation
Some lines were incorrectly fixed, so a manual check was done on all changes related to string formatting
import modifications were not checked manually

### CODE REVIEW

File: vulture_os/toolkit/updates/v1.0.10/0_reload_api_frontends_conf.py [hunk 1]
```diff
@@ -62,6 +61,6 @@
         node.api_request("services.rsyslogd.rsyslog.restart_service")
         if not api_res.get("status"):
             print("Error while restarting rsyslog: "
-                    "{}.".format(frontend.name, api_res.get("message")))
+                    "{}.".format(api_res.get("message")))
```
Comment (ELDiablO59152) [inline vulture_os/toolkit/updates/v1.0.10/0_reload_api_frontends_conf.py:64]: It could be an undesired change here, check also 5 liines above.

Comment (frikilax) [reply vulture_os/toolkit/updates/v1.0.10/0_reload_api_frontends_conf.py:64]: Yep, that change it valid!

File: vulture_os/system/cluster/models.py [hunk 2]
```diff
@@ -490,7 +489,6 @@ def get_backends_enabled(self):
         """ Return all tuples (family, proto, ip, port) for each enabled Backends on this node """
         from applications.backend.models import Backend
         # !!! REQUIRED BY listener_set !
```
Comment (ELDiablO59152) [inline vulture_os/system/cluster/models.py:491]: Same as bellow.

File: vulture_os/system/cluster/models.py [hunk 3]
```diff
@@ -379,9 +379,8 @@ def get_listeners_enabled(self):
     @property
     def get_forwarders_enabled(self):
         """ Return all tuples (family, proto, ip, port) for each LogForwarders """
-        from applications.logfwd.models import LogOM, LogOMRELP, LogOMHIREDIS, LogOMFWD, LogOMElasticSearch, LogOMMongoDB, LogOMKAFKA
+        from applications.logfwd.models import LogOM, LogOMRELP, LogOMFWD, LogOMElasticSearch, LogOMMongoDB, LogOMKAFKA
         # !!! REQUIRED BY listener_set !
```
Comment (ELDiablO59152) [inline vulture_os/system/cluster/models.py:383]: I guess you can delete this line also, it may have been a limitation in the past.

**Review by ELDiablO59152 — COMMENTED** [review]
I spotted these comment lines in the past, they can now be removed.

File: vulture_os/toolkit/updates/v1.0.10/0_reload_api_frontends_conf.py (hunk 1 above)
Comment (ELDiablO59152) [reply vulture_os/toolkit/updates/v1.0.10/0_reload_api_frontends_conf.py:64]: Seems fair.

File: vulture_os/system/cluster/models.py [hunk 4]
```diff
@@ -379,9 +379,8 @@ def get_listeners_enabled(self):
     @property
     def get_forwarders_enabled(self):
         """ Return all tuples (family, proto, ip, port) for each LogForwarders """
-        from applications.logfwd.models import LogOM, LogOMRELP, LogOMHIREDIS, LogOMFWD, LogOMElasticSearch, LogOMMongoDB, LogOMKAFKA
+        from applications.logfwd.models import LogOM, LogOMRELP, LogOMFWD, LogOMElasticSearch, LogOMMongoDB, LogOMKAFKA
```
Comment (ELDiablO59152) [inline vulture_os/system/cluster/models.py:382]: I am not truely ok with this change, we need this import but it is declared also at line 47.
A switch would be better.

Comment (frikilax) [reply vulture_os/system/cluster/models.py]: Corrected : imports are back to the top of the file

File: vulture_os/system/cluster/models.py (hunk 3 above)
Comment (frikilax) [reply vulture_os/system/cluster/models.py]: done

**Review by ELDiablO59152 — APPROVED** [review]
I approve.

File: vulture_os/system/cluster/models.py (hunk 2 above)
Comment (frikilax) [reply vulture_os/system/cluster/models.py:491]: import moved back to the beginning of the file

**Review by ELDiablO59152 — APPROVED** [review]

**Review by ELDiablO59152 — APPROVED** [review]

**Pull request merged**